sound/soc/sof/control.c

```diff
@@ -302,10 +302,13 @@ int snd_sof_bytes_ext_put(struct snd_kcontrol *kcontrol,
 		return -EFAULT;
 
 	length = header[1];
-	dev_dbg(sdev->dev, "size of bytes put data is %d\n", length);
+	if (max_length > be->max)
```

Comment:
We probably want a dev_warn() here stating that userspace is sending too much data and it has been truncated